common/data_refinery_common/models/test_organisms.py

```diff
@@ -118,18 +118,18 @@ def tearDown(self):
 
     @patch('data_refinery_common.models.organism.requests.get')
     def test_cached_names_are_found(self, mock_get):
-        Organism.objects.create(name="HOMO SAPIENS",
```

Comment:
Just curious, is there any advantage using underscore instead of space in organism name?

Reply:
We use this information to store the information on the disk, and I don't like having spaces in Unix filenames as it can break some scripting tools.